.github/workflows/main.yml

```diff
@@ -112,7 +107,7 @@ jobs:
     strategy:
       matrix:
         os: [ "ubuntu-22.04", "macos-13", "windows-2022" ]
-        dotnet: [ '6.0.x', '7.0.x', '8.0.x' ]
```

Comment:
Is there any harm in leaving `7.0.x`? Feels to me like it might be good to include all supported versions. Or maybe we want to remove `6.0.x` instead, as it will be going out of support shortly.

Reply:
7.0 is out of support as of May this year

Reply:
Whoops, guess I need to get with the times!